src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java

```diff
@@ -28,13 +28,104 @@
 import com.puppycrawl.tools.checkstyle.api.DetailAST;
 import com.puppycrawl.tools.checkstyle.api.TokenTypes;
 
-/**
- * Checks the npath complexity against a specified limit (default = 200).
- * The npath metric computes the number of possible execution paths
- * through a function. Similar to the cyclomatic complexity but also
- * takes into account the nesting of conditional statements and
- * multi-part boolean expressions.
+/**<p>
```

Comment:
Shouldn't the `<p>` start on the next line?

Reply:
I have no strict preference

Reply:
In PR we have a new Check that will enforce strict layout.

Reply:
done